dataactbroker/routeUtils.py

```diff
@@ -28,6 +28,8 @@ def run_instance_function(accountManager, accountFunction, getSystemEmail = Fals
                 else:
                     # Currently no functions with user but not credentials flag
                     raise ValueError("Invalid combination of flags to run_instance_function")
+            elif(addTrueFlag):
```

Comment:
When this is true it passes "True" into the function being called as the first argument.  This is used for the warning reports route to use the same function as the error reports route with an extra argument to get warnings.  If it would be clearer, we could replace this by having the route call a separate function in fileHandler that would call getErrorReportURLsForSubmission with the warning flag.